c++/src/kj/async.c++

```diff
@@ -2854,7 +2850,7 @@ Promise<void> yield() {
       output.as<_::Void>() = _::Void();
     }
     void tracePromise(_::TraceBuilder& builder, bool stopAtNextEvent) override {
-      builder.add(reinterpret_cast<void*>(&kj::evalLater<DummyFunctor>));
+      builder.add(reinterpret_cast<void*>(&kj::yield));
```

Comment:
Not entirely sure about this and the one below – I think this is safe but referencing yield within yield feels slightly too clever.

Reply:
Yes this is correct, the goal is only to get an address which can be put into a stack trace such that when it's later symbolicated it tells people what this promise node means.

You can also delete the definition of `DummyFunctor` now.

Reply:
Oh you already deleted it, great!